frontend/js/DataProcessing/FileProcessing.js

```diff
@@ -1,5 +1,5 @@
 var indexStack = new Array();
-
+var functionModels = new Map();
```

Comment:
might be better to store in other file

Reply:
I think it should be there because data that this map contains has to be processed. which is done in the same file.